data/movie/Stranded (2013)/blu-ray-2013/disc01-summary.txt

```diff
@@ -0,0 +1,37 @@
+Name: Stranded
+Source file name: 00002.mpls
+Duration: 1:27:51
+Chapters count: 8
+Size: 18.8 GB
+Segment count: 1
+Segment map: 0
+Type: MainMovie
+File name: Stranded.mkv
+
+Name: Trailers
+Source file name: 00009.mpls
+Duration: 0:04:40
+Chapters count: 2
+Size: 713.9 MB
+Segment count: 2
+Segment map: 3,4
+Type: Trailer
+File name: Trailers.mkv
+
+Name: Stranded - FX
+Source file name: 00001.m2ts
+Duration: 0:06:15
+Size: 890.3 MB
+Segment count: 1
+Segment map: 1
+Type: Extra
+File name: Stranded - FX.mkv
+
+Name: Stranded - Behind the Scenes
```

Comment:
Same comment here - should this just be called "Behind the Scenes"?

Reply:
I can do that. I suppose most discs probably won't have extras like that for more than one movie.